locales/fr.json

```diff
@@ -246,17 +247,37 @@
   "You will sync when you are both online.": "Vous serez synchronisé lorsque vous serez tous les deux en ligne.",
   "Please wait...": "Veuillez patienter…",
   "Create DHT Invite": "Créer une invitation DHT",
-  "Redeeming...": "Redeeming…",
+  "Redeeming...": "Utilisation…",
   "Add tags here": "Ajouter des étiquettes ici",
   "Save": "Enregistrer",
   "Participating": "Où j'ai participé",
-  "All Threads": "Tous les sujets",
-  "Threads Started By You": "Sujets que j'ai lancés",
+  "All Threads": "Tous les fils de discussion",
+  "Threads Started By You": "Fils que j'ai lancés",
   "Notification Options": "Options de notification",
   "Include \"Participating\" tab in navigation bar": "Ajouter l'onglet \"auxquels je participe\" dans la barre de navigation",
   "Blocking": "Blocage",
   "Automatically delete messages from blocked authors. This is irreversible and will cause problems with clients that share the database but do not support deleted messages. Enable at your own risk!": "Supprimer automatiquement les message des utilisateurs bloqués. Ceci est irréversible et causera des des problèmes avec les clients qui partagent la base de données mais ne prennent pas en charge les message supprimés. Activez cette option à vos risques et périls!",
   "light": "clair",
   "dark": "sombre",
+  "Clear Draft": "Supprimer Brouillon",
+  "Possible connections": "Connexions possibles",
+  "Connections": "Connexions",
+  "Search for word, @feedId, #channel or %msgId\nYou can also add author:@id or is:private for more filtering": "Rechercher mot, @id, #salon ou %msgId\nVous pouvez aussi ajouter author:@id ou is:private pour mieux filtrer",
+  "Connect": "Connecter",
+  "Click to unlike": "Cliquer pour ne plus aimer",
+  "made changes to": "a changé(e)",
+  "forked from parent thread ": "dérivé du fil parent ",
+  "(ignored)": "(ignoré)",
+  "(listening)": "(à l'écoute)",
+  "Click for options to block syncing with this person and/or hide their posts": "Cliquer pour les options de blocage de synchronisation avec cette personne et/ou cacher ses messages",
+  "Options": "Options",
+  "You have chosen to publicly block this person.": "Vous avez choisi de bloquer publiquement cette personne.",
+  "You have chosen to privately ignore this person.": "Vous avez choisi en privé d'ignorer cette personne.",
+  "No new messages will be downloaded. Existing messages will be hidden.": "Aucun nouveau message ne sera téléchargé. Les messages existants seront cachés.",
+  "el": "Grec",
+  "eo": "Esperanto",
+  "nl": "Néerlandais",
+  "zh-TW": "Taïwanais",
+  "Font Family": "Font Family",
```

Comment:
Did you leave this one on purpose?

Reply:
Which one ? I just take what you did and complete the language. Font Family is useless but as I do not know if remove it can cause a side effect. I leave it as it is.

Reply:
"Font Family" is used in the settings screen: 

![Screenshot from 2020-04-26 23-42-05](https://user-images.githubusercontent.com/189241/80320594-8e39cf00-8817-11ea-96e7-fdc2137ad4e7.png)

Reply:
oops, sorry, I will translate it.